src/test/java/com/codecool/terraformingmarsforum/service/DAO/memory/CommentMemoryTest.java

```diff
@@ -0,0 +1,22 @@
+package com.codecool.terraformingmarsforum.service.DAO.memory;
+
+import com.codecool.terraformingmarsforum.model.Comment;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CommentMemoryTest {
+
+    @Test
+    void add_CreateNewCommit_PresentInMemory() {
```

Comment:
I bet you meant comment not commit (but I will merge it bc its just a test method name)

Reply:
Okay, next time please do not approve my pull request if you find things like that :)